app-frontend/src/app/pages/projects/edit/advancedcolor/adjust/adjust.controller.js

```diff
@@ -43,56 +43,35 @@ export default class ProjectsColorAdjustController {
             showTicks: 1
         };
 
-        let allGamma = ['redGamma', 'greenGamma', 'blueGamma'];
-
         this.redGammaOptions = Object.assign({
             id: 'redGamma',
-            onEnd: (id, val) => {
-                this.onFilterChange(
-                    this.gammaLinkToggle ? allGamma : 'redGamma', val, this.redGammaOptions
-                );
-            }
+            onEnd: (id, val) => this.onGammaFilterChange(val)
         }, baseGammaOptions);
         this.greenGammaOptions = Object.assign({
             id: 'greenGamma',
-            onEnd: (id, val) => {
-                this.onFilterChange(
-                    this.gammaLinkToggle ? allGamma : 'greenGamma', val, this.greenGammaOptions
-                );
-            }
+            onEnd: (id, val) => this.onGammaFilterChange(val)
         }, baseGammaOptions);
         this.blueGammaOptions = Object.assign({
             id: 'blueGamma',
-            onEnd: (id, val) => {
-                this.onFilterChange(
-                    this.gammaLinkToggle ? allGamma : 'blueGamma', val, this.blueGammaOptions
-                );
-            }
+            onEnd: (id, val) => this.onGammaFilterChange(val)
         }, baseGammaOptions);
 
         this.saturationOptions = Object.assign({
             id: 'saturation',
-            onEnd: (id, val) => {
-                this.onFilterChange(id, val, baseSaturationOptions);
-            }
+            onEnd: () => this.onFilterChange()
         }, baseSaturationOptions);
 
         this.alphaOptions = Object.assign({
             id: 'alpha',
-            onEnd: (id, val) => this.onFilterChange(id, val, this.alphaOptions)
+            onEnd: () => this.onFilterChange()
```

Comment:
This part is fine, but the object that this pulls from, `alphaOptions` has a `ceil` that doesn't match the maximum value that is set on the `input` in the template. (`ceil` is 2, the input's max is 1). I don't think this is anything you did, but it's probably a good point to make the fix.

The beta input doesn't match it's `ceil` either.